include/GafferScene/SceneAlgo.h

```diff
@@ -78,8 +78,30 @@ void matchingPaths( const Gaffer::IntPlug *filterPlug, const ScenePlug *scene, P
 /// As above, but specifying the filter as a PathMatcher.
 void matchingPaths( const PathMatcher &filter, const ScenePlug *scene, PathMatcher &paths );
 
-/// Calls a functor on all paths in the scene
-/// The functor must take ( const ScenePlug*, const ScenePlug::ScenePath& ), and can return false to prune traversal
+/// Invokes the Functor at every location in the scene,
```

Comment:
I checked gaffer and thought this was safe (perhaps barring minor performance implications) ones but you're right about other usages at IE requiring a single functor to accumulate results.

So I'll keep parallelTraverse in place and just add parallelProcessLocations